File: PDFaCodigoJavaCSharpRustPython/PA_StopWatch/CSharpCode/ejecutar.cs
using System;
using System.Windows;

namespace CSharpCode {
    public class ejecutar {
        [STAThread]
        public static void Main(string[] args) {
            Application app = new Application();
            Mstopwatch model = new Mstopwatch();
            Vstopwatch view = new Vstopwatch();
            new Cstopwatch(model, view);
            app.Run(view); // Inicia el JFrame equivalente
        }
    }
}

File: PDFaCodigoJavaCSharpRustPython/PA_StopWatch/PythonCode/ejecutar.py
from Mstopwatch import Mstopwatch
from Vstopwatch import Vstopwatch
from Cstopwatch import Cstopwatch

if __name__ == "__main__":
    model = Mstopwatch()
    view = Vstopwatch()
    Cstopwatch(model, view)
    view.mainloop()